src/Security/SignedRequestAuthenticator.php

```diff
@@ -14,17 +14,19 @@
 use Twig\Error\LoaderError;
 use Twig\Error\RuntimeError;
 use Twig\Error\SyntaxError;
+use App\Entity\AppInfo;
+use Doctrine\Persistence\ManagerRegistry;
 
 class SignedRequestAuthenticator extends AbstractAuthenticator
 {
 
     private Environment $twig;
-    private string $clientSecret;
+    private ManagerRegistry $doctrine;
```

Comment:
You don't need to get the Managerregistry here, just inject the repository itself ^^